server/graphql/v2/mutation/RootMutations.ts

```diff
@@ -251,4 +253,38 @@ export default {
       return moveExpenses(req, expenses, destinationAccount);
     },
   },
+  setTaxForm: {
+    type: new GraphQLObjectType({
+      name: 'SetTaxFormResult',
+      fields: {
+        success: { type: GraphQLBoolean },
+      },
+    }),
+    description: '[Root only] A mutation to set the tax from for an account.',
+    args: {
+      account: {
+        type: new GraphQLNonNull(AccountReferenceInput),
+        description: 'Reference to the Account the tax form should be set.',
+      },
+      taxFormLink: {
+        type: new GraphQLNonNull(GraphQLNonEmptyString),
+        description: 'The tax from link.',
+      },
+      year: {
+        type: new GraphQLNonNull(GraphQLInt),
+        description: 'The tax form year.',
+      },
+    },
+    async resolve(_, args, req) {
+      checkRemoteUserCanRoot(req);
+      // Always enforce 2FA for root actions
+      await twoFactorAuthLib.validateRequest(req, { requireTwoFactorAuthEnabled: true });
+
+      const account = await fetchAccountWithReference(args.account, { throwIfMissing: true });
+
+      await setTaxForm(account, args.taxFormLink, args.year);
+
+      return { success: true };
```

Comment:
`setTaxForm` can return false if the document doesn't exist

```suggestion
      return { success: setTaxForm(account, args.taxFormLink, args.year) };
```